### PR TITLE
Update DeveloperGuide

### PR DESCRIPTION
**Pull request by wesho1107**
- Update UML Class Diagram
- Add Activity Diagram

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -461,7 +461,7 @@ The `UpdateTaskProgressCommand` will call `setTask` in `Model` to replace the ex
 Step 4:
 The `UpdateTaskProgressCommand` will call its own `updateFilteredTaskList` method to update the model's filter and display all the tasks to the user.
 
-Step 4:
+Step 5:
```
Comment (Respirayson) [inline docs/DeveloperGuide.md:530]: oops my bad good catch!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -478,6 +478,82 @@ The `UpdateTaskProgressCommand` then continues its execution as defined by [this
     * Cons: Reduced maintainability as state of object can keep changing throughout the code.
 
 
+
+#### Create Consultation Feature
+This section explains the implementation of the Create Consultation feature via the `createconsult` command.
+The `CreateConsultCommand` creates a `Consultation` and adds it into the Consultation List in the application.
+There is are multiple compulsory field: date, time and name of students.
```
Comment (Respirayson) [inline docs/DeveloperGuide.md]: There are *

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -478,6 +478,82 @@ The `UpdateTaskProgressCommand` then continues its execution as defined by [this
...
+* **Alternative 2:** Store the model in the `CreateConsultCommand` itself.
+    * Pros: Easier to debug.
+    * Cons: The `CreateConsultCommand` might be able to call other methods in the model.
+
+
+
+#### Add To Consultation Feature
+This section explains the implementation of the Add To Consultation feature via the `addtoconsult` command.
+The `AddToConsultCommand` adds a new student to the consultation identified using the Index.
+There is are 2 compulsory field, which are the Index of the consultation to add student into, and the name of the student.
```
Comment (Respirayson) [inline docs/DeveloperGuide.md]: There are*

File: docs/diagrams/ConsultationClassDiagram.puml [hunk 4]
```diff
@@ -1,20 +1,36 @@
-@startuml
+@startuml Consultation
 hide circle
 skinparam classAttributeIconSize 0
 
-Class Consultation {
+class Consultation {
```
Comment (Respirayson) [inline docs/diagrams/ConsultationClassDiagram.puml]: I thought class was supposed to be capitalised? With Reference to ModelClassDiagram

**Review by Respirayson — COMMENTED** [review]
small nits here and there

File: docs/diagrams/ConsultationClassDiagram.puml (hunk 4 above)
Comment (wesho1107) [reply docs/diagrams/ConsultationClassDiagram.puml]: oh, I was referencing to dinghan's class diagram, must have been mistaken, will change both for standardisation!

**Review by ldinghan — APPROVED** [review]
LGTM!

File: docs/images/DeleteTaskSequenceDiagram.png
Comment (raydenlim) [inline docs/images/DeleteTaskSequenceDiagram.png:1]: All the words look bold in this new image. Not sure if thats an issue :octocat:

Comment (raydenlim) [reply docs/images/DeleteTaskSequenceDiagram.png:1]: Same for the other diagrams

Comment (wesho1107) [reply docs/images/DeleteTaskSequenceDiagram.png:1]: I'm not sure, I saved it as a diagram as suggested from IntelliJ.

**Pull request merged**